docs/Learning-Environment-Design-Agents.md

```diff
@@ -112,6 +109,8 @@ of data to your observation vector. You can add Integers and booleans directly t
 the observation vector, as well as some common Unity data types such as `Vector2`,
 `Vector3`, and `Quaternion`.
 
+#### One-hot encoding categorical information
+
```

Comment:
(sorry, can't comment directly on the line)
VectorSensor doesn't have an overload for one-hot, it has its own AddOneHotObservation() method.